docs/koji.md

```diff
@@ -0,0 +1,313 @@
+# Koji
+
+[Koji](https://pagure.io/koji/) is an RPM build and tracking system. The Foreman project Koji instancne lives at [Foreman Koji](http://koji.katello.org/koji/).
+
+ * Using Koji as a user
```

Comment:
This will turn into a table of contents once the structure is settled

Reply:
I'd prefer to use some proper documentation tool that automatically generates it. We can use sphinx in combination with readthedocs.io. We already use it for obal @ https://obal.readthedocs.io/en/latest/. While we use .rst there, it can use .md as well. When using .rst you can also include content from other files if desired, which can allow us to reuse the Puppet code as documentation.